### PR TITLE
4.get articles

### PR DESCRIPTION
**Pull request by K-Othman**

### CODE REVIEW

File: __tests__/news.test.js [hunk 1]
```diff
@@ -29,3 +29,78 @@ describe.only("1. GET /api/topics", () => {
       });
   });
 });
+
+describe.only("4. GET /api/articles", () => {
```
Comment (RichardNeat) [inline __tests__/news.test.js]: Please remove .onlys before submitting. We can't be sure that all your tests are passing through husky like this.

File: __tests__/news.test.js [hunk 2]
```diff
@@ -29,3 +29,78 @@ describe.only("1. GET /api/topics", () => {
...
+  test("status:200, responds with an array of articles objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
```
Comment (RichardNeat) [inline __tests__/news.test.js:43]: Consider using a more flexible jest matcher here, if the test database is ever updated we don't want to have to go back and edit our tests.

File: __tests__/news.test.js [hunk 3]
```diff
@@ -29,3 +29,78 @@ describe.only("1. GET /api/topics", () => {
...
+        created_at: "2020-01-07T14:08:00.000Z",
+        votes: 0,
+        article_id: 1,
+        comment_count: 11,
+      },
+    ];
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .send(output)
```
Comment (RichardNeat) [inline __tests__/news.test.js]: I think we need to look at reworking this test, we are sending an input body but the endpoint is a GET not a POST/PATCH?

File: __tests__/news.test.js [hunk 4]
```diff
@@ -29,3 +29,78 @@ describe.only("1. GET /api/topics", () => {
...
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
```
Comment (RichardNeat) [inline __tests__/news.test.js:55]: We also need a test to see what order these come back in

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Hi could you have a look a look at the below comments regarding your testing. Once those tests are solid we can have a look at your functionality. Thanks :)

File: __tests__/news.test.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ const app = require("../app");
 const db = require("../db");
 const data = require("../db/data/test-data");
 const seed = require("../db/seeds/seed");
+const jestSorted = require("jest-sorted");
```
Comment (dav3rid) [inline __tests__/news.test.js:6]: You shouldn't need to bring in jest sorted to the test file - instead, take a look at the docs and see how you can configure your `package.json` to use it
[https://www.npmjs.com/package/jest-sorted](https://www.npmjs.com/package/jest-sorted)

**Review by dav3rid — APPROVED** [review]

**Pull request merged**